src/client/containers/BoardPageComponents/BoardAddCandidate/BoardAddCandidatePostDeadline.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './BoardAddCandidate.style.css';
+import Input from '../../../components/InputComponent/InputComponent';
+import AddButton from '../../../components/AddButton/AddButton.component';
+
+export default function BoardAddCandidate() {
```

Comment:
I think the Compponents should be named the same as the files they are in

Reply:
yes, It's done now.